client/packages/system/src/Manage/Preferences/api/usePreferencesGraphQL.ts

```diff
@@ -4,12 +4,13 @@ import {
   useQueryClient,
 } from '@openmsupply-client/common';
 import { getSdk } from './operations.generated';
+import { PREFERENCES } from './keys';
 
 export const usePreferencesGraphQL = () => {
   const { client } = useGql();
   const queryClient = useQueryClient();
   const { storeId, store } = useAuthContext();
   const api = getSdk(client);
 
-  return { api, queryClient, storeId, store };
+  return { api, queryClient, storeId, store, client, PREFERENCES };
```

Comment:
Just re-exporting key from here to avoid two ugly long imports in `usePreference`